contracts/test/TestUtils.sol

```diff
@@ -26,27 +27,29 @@ contract TestUtils {
         return signature;
     }
 
-    function createUOTestData(address sender, bytes memory proofData)
+    /// @notice Create a test data for UserOperations.
+    function createUOTestData(address sender, bytes[] memory proofs)
         public
         pure
         returns (PackedUserOperation[] memory)
     {
-        bytes memory signature = encodeSignature(proofData);
-        PackedUserOperation[] memory uOps = new PackedUserOperation[](2);
-        PackedUserOperation memory baseUO = PackedUserOperation({
-            sender: sender,
-            nonce: 0,
-            initCode: abi.encodePacked("0x"),
-            callData: abi.encodePacked("0x"),
-            accountGasLimits: bytes32("10000"),
-            preVerificationGas: 10000,
-            gasFees: bytes32(0),
-            paymasterAndData: abi.encodePacked("0x"),
-            signature: signature
-        });
+        PackedUserOperation[] memory uOps = new PackedUserOperation[](proofs.length);
+        for (uint256 i = 0; i < proofs.length; ++i) {
+            bytes memory signature = encodeSignature(proofs[i]);
+            PackedUserOperation memory uo = PackedUserOperation({
```

Comment:
Nit: Either remove or uncomment.

Reply:
I think I did while you were writing this comment :)